hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/clientrm/TestFederationClientInterceptor.java

```diff
@@ -293,17 +288,15 @@ public void testSubmitApplicationEmptyRequest()
       interceptor.submitApplication(null);
       Assert.fail();
```

Comment:
LambdaTestUtils#intercep

Reply:
Thank you for helping to review the code, I will fix it.